extension/persistence/eclipselink/build.gradle.kts

```diff
@@ -32,6 +32,7 @@ dependencies {
   implementation(libs.eclipselink)
   implementation(platform(libs.dropwizard.bom))
   implementation("io.dropwizard:dropwizard-jackson")
+  implementation("org.glassfish.hk2:hk2-api:3.0.6")
```

Comment:
Why does EclipseLink need an explicit dependency on HK2? Shouldn't is be sufficient to put HK2 in the DW application?

Reply:
This was from when I had the MetaStoreManagerFactory extend the Factory interface - it's unnecessary now. I'll remove it.

Reply:
it's still here? ;)